synthetics/agents_distance_filter.go

```diff
@@ -0,0 +1,21 @@
+package synthetics
+
+import (
+	geo "github.com/kellydunn/golang-geo"
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/synthetics"
+)
+
+func filterAgentsByDistance(agents []synthetics.V202101beta1Agent,
+	lat float64, long float64, distConstr float64) []synthetics.V202101beta1Agent {
+	var filteredAgents []synthetics.V202101beta1Agent
+	referencePoint := geo.NewPoint(lat, long)
+	for _, agent := range agents {
+		agentCoordinates := geo.NewPoint(*agent.Lat, *agent.Long)
+		dist := referencePoint.GreatCircleDistance(agentCoordinates)
+		if dist < distConstr {
```

Comment:
We can do `dist <= distConstr`

Reply:
Addressed: 5be5d29456a2413ec9afec8653955a0cab7196e2